src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicMobsMobTag.java

```diff
@@ -387,6 +400,171 @@ public static void registerTags() {
             }
             return result;
         });
+
+        // <--[tag]
+        // @attribute <MythicMobsMobTag.ai_goal_selectors>
+        // @returns ListTag
+        // @plugin Depenizen MythicMbos
+        // @description
+        // Returns a ListTag of the MythicMob's AIGoalSelectors.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "ai_goal_selectors", (attribute, object) -> {
+            ListTag list = new ListTag();
+            for (String goal : object.getMobType().getAIGoalSelectors()) {
+                list.addObject(new ElementTag(goal, true));
+            }
+            return list;
+        });
+
+        // <--[tag]
+        // @attribute <MythicMobsMobTag.ai_target_selectors>
+        // @returns ListTag
+        // @plugin Depenizen MythicMbos
+        // @description
+        // Returns a ListTag of the MythicMob's AITargetSelectors.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "ai_target_selectors", (attribute, object) -> {
+            ListTag list = new ListTag();
+            for (String target : object.getMobType().getAITargetSelectors()) {
+                list.addObject(new ElementTag(target, true));
+            }
+            return list;
+        });
+
+        // <--[tag]
+        // @attribute <MythicMobsMobTag.damage_modifiers>
+        // @returns MapTag
+        // @plugin Depenizen MythicMobs
+        // @description
+        // Returns a MapTag of the MythicMob's damage modifiers.
+        // -->
+        tagProcessor.registerTag(MapTag.class, "damage_modifiers", (attribute, object) -> {
+            MapTag result = new MapTag();
+            for (Map.Entry<String, Double> entry : object.getMobType().getDamageModifiers().entrySet()) {
+                result.putObject(entry.getKey(), new ElementTag(entry.getValue().toString()));
+            }
+            return result;
+        });
+
+        // <--[tag]
+        // @attribute <MythicMobsMobTag.faction>
+        // @returns ElementTag
+        // @plugin Depenizen MythicMobs
+        // @description
+        // Returns the mob's faction.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "faction", (attribute, object) -> {
+            return new ElementTag(MythicMobsBridge.getFaction(object.getMob()), true);
+        });
+
+        // <--[tag]
+        // @attribute <MythicMobsMobTag.children>
+        // @returns ListTag(EntityTag)
+        // @plugin Depenizen MythicMobs
+        // @description
+        // Returns the mob's children.
+        // @mechanism MythicMobsMobTag.add_child
```

Comment:
obscure minor meta rule, but: put `@mechanism` before the description rather than after